examples/pytorch/pointcloud/bipointnet/README.md

```diff
@@ -11,9 +11,18 @@ This project is the official implementation of our accepted ICLR 2021 paper *BiP
 ### How to Run
 
 ```shell script
-python train_cls.py
+python train_cls.py --model ${MODEL}
 ```
 
+Here, `MODEL` have two choices: `bipointnet`  and  `bipointnet2_ssg`
+
+# Performance
+
+| Task              | Dataset    | Metric   | Score - DGL (Adam) |
+| ----------------- | ---------- | -------- | ------------------ |
+| Classification    | ModelNet40 | Accuracy | 89.4               |
```

Comment:
How do we compare with the numbers in the paper? Would there be an explanation of the gap?